Formula/lsof.rb

```diff
@@ -16,23 +16,29 @@ class Lsof < Formula
   keg_only :provided_by_macos
 
   def install
-    ENV["LSOF_INCLUDE"] = "#{MacOS.sdk_path}/usr/include"
+    os = "linux"
+    on_macos do
+      ENV["LSOF_INCLUDE"] = "#{MacOS.sdk_path}/usr/include"
+
+      # Source hardcodes full header paths at /usr/include
+      inreplace %w[
+        dialects/darwin/kmem/dlsof.h
+        dialects/darwin/kmem/machine.h
+        dialects/darwin/libproc/machine.h
+      ], "/usr/include", "#{MacOS.sdk_path}/usr/include"
+
+      os = "darwin"
+    end
+
     ENV["LSOF_CC"] = ENV.cc
     ENV["LSOF_CCV"] = ENV.cxx
 
-    # Source hardcodes full header paths at /usr/include
-    inreplace %w[
-      dialects/darwin/kmem/dlsof.h
-      dialects/darwin/kmem/machine.h
-      dialects/darwin/libproc/machine.h
-    ], "/usr/include", "#{MacOS.sdk_path}/usr/include"
-
     mv "00README", "README"
-    system "./Configure", "-n", "darwin"
+    system "./Configure", "-n", os
 
     system "make"
     bin.install "lsof"
-    man8.install "lsof.8"
+    man8.install "Lsof.8"
     prefix.install_metafiles
```

Comment:
```suggestion
```
Since we're here already let's remove this now as it's no longer needed.

Reply:
If you say so :) I have no clue what this does/did.

Reply:
Installs the metafiles, obviously 😄

Reply:
I mean, it installed metafiles, and now the metafiles are not needed. That doest not tell me why this needs a change 😄

Reply:
Oh it's no longer needed because `brew` now installs the metafiles without being told.